packages/common/src/restHelpers.ts

```diff
@@ -1719,6 +1721,36 @@ export function updateItem(
   });
 }
 
+/**
+ * Updates a group.
+ *
+ * @param groupInfo The base info of a group; note that this content will be serialized, which doesn't work
+ * for binary content
+ * @param authentication Credentials for request
+ * @param additionalParams Updates that are put under the `params` property, which is not serialized
+ * @return
```

Comment:
What is the return, please?  And can we put a real type in instead of "any" for additionalParams and the return?